examples/gnn_node.py

```diff
@@ -47,6 +47,9 @@
 dataset: RelBenchDataset = get_dataset(name=args.dataset, process=True)
 task: NodeTask = dataset.get_task(args.task, process=True)
 
+# Visualize the db
+dataset.db.visualize(f"{args.dataset}.png")
```

Comment:
Not sure if we need this in example code.

Reply:
Will remove